Makefile

```diff
@@ -36,3 +36,5 @@ uninstall:
 lint:
 	go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@latest
 	${GOPATH}/bin/golangci-lint run --deadline=3m --timeout=3m ./... # Run linters
+mockgen:
+	mockgen -source=pkg/sync/http_sync.go -destination=pkg/sync/mock/http.go -package=syncmock
```

Comment:
Where does this binary come from?

Reply:
Added a comment above this line for clarification

Reply:
<img width="1166" alt="Screenshot 2022-07-29 at 14 58 08" src="https://user-images.githubusercontent.com/1235925/181776113-e879e6ce-64cb-4baf-bb31-f2adcb0e3984.png">

Reply:
Can you add this as a target to auto install? We prefer to have dependencies carried in the Makefile rather than outside the project.

Reply:
Definitely agree with this.

Reply:
Good shout, done :)